### ISSUE TITLE
Error message when playing any animation playback track

### ISSUE THREAD

**Issue by Gnumaru** [godotengine/godot#72374]
### Godot version

4.0.beta16

### System information

windows 10

### Issue description

whenever an animation starts playing and there is an animation track in it with at least one key, the seemingly innocuous error message gets printed:
```
ERROR: Condition "!playback.current.from" is true.
   at: seek (scene/animation/animation_player.cpp:1784)
```
It apparently doesn't cause any kind of erroneous behavior, and the other animation player seems to play the animation as expected.

### Steps to reproduce

see attached video:

https://user-images.githubusercontent.com/5783414/215481088-a0da14c0-fa1a-4921-b351-66257adb9a3a.mp4

### Minimal reproduction project

[PlayingAnimationTracksPrintsAnError.zip](https://github.com/godotengine/godot/files/10535868/PlayingAnimationTracksPrintsAnError.zip)

### Work Around

Instead of using animation playback tracks, the user can use method call tracks and call the play method in the desired animation player

**Comment by akien-mga** [comment]
Seems to be a regression, the bug reproduces in beta 9 to 16, but not in beta 8 and earlier. Animation PRs in that timeframe: https://github.com/godotengine/godot/pulls?q=is%3Apr+merged%3A2022-12-09..2022-12-18+is%3Amerged+sort%3Acreated-asc+milestone%3A4.0+label%3Atopic%3Aanimation

CC @TokageItLab

**Comment by eh-jogos** [comment]
Also, forgot to mention in the comment above, but the error does not happen if the "base scene" AnimationPlayer has no anim library, in that case everything works fine, with no error. In the MRP you can delete the anim library from the Base Scene and see that it will stop the seek error message from appearing when playing the "Inherited" scene

**Comment by TokageItLab** [comment]
May be fixed by #72380.
